packages/prettier-plugin-java/src/cst-printer.js

```diff
@@ -73,6 +77,17 @@ class CstPrettierPrinter extends BaseJavaCstVisitor {
         return "";
       }
 
+      if (ctx.ignore) {
+        try {
+          return buildOriginalText(retrieveNodesToken(ctx));
+        } catch (e) {
+          // eslint-disable-next-line no-console
+          console.error(
+            "There might be a problem with prettier-ignore, check cst-printer.js"
```

Comment:
Maybe change the message for end-users.

Reply:
This should not be happening, if an error is thrown this is probably due to a forgotten case or e.g.

Reply:
Yes, so maybe put something like: "Please report an issue on..."

Reply:
Yes, will do :+1: